portfolio/src/main/webapp/style.css

```diff
@@ -77,7 +83,7 @@ html{
 
 #projects {
   background-color: #343a40;
-  height: 60vh;
+  height: 50vh;
```

Comment:
This is totally optional:
Since you specifically mentioned about feedback on CSS, it's better to automatically adapt to the content than needing to fudge with CSS for every change.

For example, you can give min-height to this section and adjust the margin and padding...something like:
```
min-height: 200px;
margin-top: 100px;
padding-bottom: 50px;
```
Now, you can change your projects and descriptions and you won't need to touch the CSS.

**Long:**
![image](https://user-images.githubusercontent.com/22455214/83553334-0b242c80-a4d9-11ea-995b-05c91ae68a1d.png)

**Short:**
![image](https://user-images.githubusercontent.com/22455214/83553352-111a0d80-a4d9-11ea-8224-89c702d91d58.png)